XCUITests/SmokeXCUITests.xctestplan

```diff
@@ -13,6 +13,7 @@
   },
   "testTargets" : [
     {
+      "parallelizable" : true,
```

Comment:
Not sure if we want to enable this... test execution will be reduced around ~7min but I'm afraid there are going to be some intermittents due to the parallel testing as [in this build ](https://app.bitrise.io/build/7cf5d010cee30d2e#?tab=log)and also locally... wdyt @AaronMT?

Reply:
I don't know anything about this.

Reply:
It usually happens that some tests are flaky when running this way... if we enable this we should be more agressive with disabling/fixing those tests... Maybe not good timing for this.. we can always try...